src/pages/item/_.vue

```diff
@@ -285,7 +291,10 @@
       },
       ...mapGetters({
         shareUrl: 'http/canonicalUrl'
-      })
+      }),
+      multilingualItemEnabled() {
+        return this.$config.app.features.multilingualItem;
+      }
     },
 
     mounted() {
```

Comment:
bellow in this file 
`watchQuery: ['metadataLang'],`
will need to be replaced with:
```
watch: {
  '$route.query.metadataLang': '$fetch'
},
```

To trigger the re-fetching of new metadata when the toggle is used..

That still doesn't quite update the whole UI properly though, I'm not sure but probably because metadataLang isn't a computed property, but just read from the URL query.

Reply:
As we also need to call asyncData I'm leaving this as is.